images/win/scripts/Installers/Validate-AWS-CLI-SM.ps1

```diff
@@ -0,0 +1,14 @@
+################################################################################
+##  File:  Validate-AWS-CLI-SM.ps1
+##  Desc:  Validate aws cli session manager
+################################################################################
+
+$sessionMessage = session-manager-plugin
+
+if ($sessionMessage -Match "*plugin was installed successfully*") {
+    Write-Host "$sessionMessage"
```

Comment:
Can we put `Write-Host "$sessionMessage"` before `if`. So we won't need to duplicate it.
Also I believe `exit 0` doesn't make more sense.

Reply:
Yes, sure. Should we totally remove if check because if command not found it will fail.

Reply:
Done.